src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/moveByNodes/GeneratePath.java

```diff
@@ -0,0 +1,32 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.moveByNodes;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeBase.SystemsState;
+import edu.greenblitz.tobyDetermined.Nodesssss.NonCollidingNodeSystemAlgorithm.AStar;
+import edu.greenblitz.utils.GBCommand;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+import java.util.LinkedList;
+import java.util.function.Supplier;
+
+import static edu.greenblitz.tobyDetermined.Nodesssss.NodeBase.CreateCurrents.system1CurrentNode;
+
+public class GeneratePath implements Supplier<Command> {
```

Comment:
Try finding a more descriptive name for the class

Reply:
I didn't found a better name and I think its a fair name cause all the class is doing is generating the command path and return it, butt if you have a better name idea you can suggest.

Reply:
My point this class has a name that sounds like a function (verb + noun)

How about PathGenerator? I think it's just a bit better

Reply:
Ok